### PR TITLE
Review fixes

### PR DESCRIPTION
**Pull request by manojmalik20**
Sorted the import order in all files
Modified removeRecentCustomStatus action not to use bindClientFunction
Added types for custom status client functions
Added containerSize prop in clear button component
Removed the check for custom status enabled from custom status emoji and added it in post_header, channel_title, channel_item
Modified getCustomStatusSelector to return undefined if custom status does not exist
Modified settings_sidebar, channel_info, channel_info_header, custom_status_modal, user_profile to handle undefined custom status
Removed the text wrapper from the custom status emoji in channel_info_header
Added check for identifying the navigation button pressed in custom status modal

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates

#### Device Information
This PR was tested on: 

#### Release Note

```release-note

```

### CODE REVIEW

File: app/screens/custom_status/custom_status_modal.tsx [hunk 1]
```diff
@@ -130,9 +136,8 @@ class CustomStatusModal extends NavigationComponent<Props, State> {
         this.setState({emoji, text});
     };
 
-    renderRecentCustomStatuses = () => {
+    renderRecentCustomStatuses = (style: any) => {
```
Comment (chetanyakan) [inline app/screens/custom_status/custom_status_modal.tsx]: Check if the type is `React.CSSProperties`?

File: app/screens/custom_status/custom_status_modal.tsx [hunk 2]
```diff
@@ -167,9 +172,8 @@ class CustomStatusModal extends NavigationComponent<Props, State> {
         );
     };
 
-    renderCustomStatusSuggestions = () => {
+    renderCustomStatusSuggestions = (style: any) => {
```
Comment (chetanyakan) [inline app/screens/custom_status/custom_status_modal.tsx]: Don't use `any`

File: app/selectors/custom_status.ts [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
     return createSelector(
         (state: GlobalState, userID?: string) => (userID ? getUser(state, userID) : getCurrentUser(state)),
         (user) => {
             const userProps = user?.props || {};
-            return userProps.customStatus && JSON.parse(userProps.customStatus);
+            if (userProps.customStatus) {
+                return JSON.parse(userProps.customStatus) || undefined;
+            }
+
+            return undefined;
```
Comment (chetanyakan) [inline app/selectors/custom_status.ts:23]: I don't think we need to explicitly return `undefined`, unless typescript complains.

```suggestion
            if (userProps.customStatus) {
                return JSON.parse(userProps.customStatus);
            }
```

**Review by chetanyakan — APPROVED** [review]

File: app/screens/custom_status/custom_status_modal.tsx (hunk 1 above)
Comment (manojmalik20) [reply app/screens/custom_status/custom_status_modal.tsx]: No actually, the type we are getting is any and taking React.CSSProperties gives error

File: app/selectors/custom_status.ts (hunk 3 above)
Comment (manojmalik20) [reply app/selectors/custom_status.ts:23]: Lint fails

**Pull request merged**